pages/apis/rest_api/pipelines.md.erb

```diff
@@ -563,6 +563,37 @@ Required scope: `write_pipelines`
 
 Success response: `204 No Content`
 
+## Add a webhook
+
+Create webhook for an existing pipeline.
+
+```bash
+curl -X POST "https://api.buildkite.com/v2/organizations/{org.slug}/pipelines/{slug}/webhook"
+```
+
+Required [request body properties](/docs/api#request-body-properties):
+
+<table>
```

Comment:
We don't need to send any params along with this request. The slugs in the path are all that's required.